semanticdb-javac/src/main/java/com/sourcegraph/semanticdb_javac/SemanticdbJavacOptions.java

```diff
@@ -23,6 +23,7 @@ public class SemanticdbJavacOptions {
   public boolean includeText = false;
   public boolean verboseEnabled = false;
   public final ArrayList<String> errors;
+  public boolean isErrorReported = false;
```

Comment:
```suggestion
  public boolean alreadyReportedErrors = false;
```

(Or maybe `finishedReportingErrors`.)

Right now, the wording makes it a little ambiguous without looking at the usage site. When I first read this, I was wondering if it meant "should we report errors at all" or "have errors been reported already."

Reply:
Good point, applied suggestion. I default to using `is` as the prefix to boolean fields but in this case I agree that `alreadyReportedErrors` is clearer.